docs/content/docs/connectors/table/jdbc.md

```diff
@@ -45,17 +45,18 @@ See how to link with it for cluster execution [here]({{< ref "docs/dev/configura
-| Trino      | `io.trino`                 | `trino-jdbc`           | [Download](https://repo1.maven.org/maven2/io/trino/trino-jdbc/)                                                                   |
-| OceanBase  | `com.oceanbase`            | `oceanbase-client`     | [Download](https://repo1.maven.org/maven2/com/oceanbase/oceanbase-client/)                                                        |
+| Driver     | Group Id                   | Artifact Id                 | JAR                                                                                                                                     |
+|:-----------|:---------------------------|:----------------------------|:----------------------------------------------------------------------------------------------------------------------------------------|
+| MySQL      | `mysql`                    | `mysql-connector-java`      | [Download](https://repo.maven.apache.org/maven2/mysql/mysql-connector-java/)                                                            |
+| Oracle     | `com.oracle.database.jdbc` | `ojdbc8`                    | [Download](https://mvnrepository.com/artifact/com.oracle.database.jdbc/ojdbc8)                                                          |
+| PostgreSQL | `org.postgresql`           | `postgresql`                | [Download](https://jdbc.postgresql.org/download/)                                                                                       |
+| Derby      | `org.apache.derby`         | `derby`                     | [Download](http://db.apache.org/derby/derby_downloads.html)                                                                             |
+| SQL Server | `com.microsoft.sqlserver`  | `mssql-jdbc`                | [Download](https://docs.microsoft.com/en-us/sql/connect/jdbc/download-microsoft-jdbc-driver-for-sql-server?view=sql-server-ver16)       |
+| CrateDB    | `io.crate`                 | `crate-jdbc`                | [Download](https://repo1.maven.org/maven2/io/crate/crate-jdbc/)                                                                         |
+| Db2        | `com.ibm.db2.jcc`          | `db2jcc`                    | [Download](https://www.ibm.com/support/pages/download-db2-fix-packs-version-db2-linux-unix-and-windows)                                 | 
+| Trino      | `io.trino`                 | `trino-jdbc`                | [Download](https://repo1.maven.org/maven2/io/trino/trino-jdbc/)                                                                         |
+| OceanBase  | `com.oceanbase`            | `oceanbase-client`          | [Download](https://repo1.maven.org/maven2/com/oceanbase/oceanbase-client/)                                                              |
+| Spanner    | `com.google.cloud`         | `google-cloud-spanner-jdbc` | [Download](https://central.sonatype.com/artifact/com.google.cloud/google-cloud-spanner-jdbc)                                            |
```

Comment:
I see the download url points to Maven like the spanner doc. How do I get the jar file?

Reply:
I have written the same URL as the official document.
- https://cloud.google.com/spanner/docs/jdbc-drivers

The Jar files can be downloaded from the following.
- https://repo1.maven.org/maven2/com/google/cloud/google-cloud-spanner-jdbc/2.27.1/
- https://repo1.maven.org/maven2/com/google/cloud/google-cloud-spanner-jdbc/2.27.1/google-cloud-spanner-jdbc-2.27.1-single-jar-with-dependencies.jar
- https://repo1.maven.org/maven2/com/google/cloud/google-cloud-spanner-jdbc/2.27.1/google-cloud-spanner-jdbc-2.27.1.jar